### DISCUSSION TITLE
Custom regex manager with github releases and multiple assets

### DISCUSSION THREAD

**Discussion by bdovaz** [renovatebot/renovate#29980] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure Pipelines

### Please tell us more about your question or problem

I am using this plugin from Kibana / OpenSearch Dashboards

https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.14.0

And in particular I am interested in the asset of each release with the format

`enhanced-table-1.14.0_osd-2.15.0.zip`

Of which as you can see there are 2 values:
- 1.14.0: release number
- 2.15.0: OpenSearch Dashboards version

And in my Dockerfile I have the following:

````dockerfile
FROM opensearchproject/opensearch-dashboards:2.15.0

RUN /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin install https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.15.0.zip
````

So I would like to know if it is possible to extract from that Dockerfile the 2 values so that renovate replaces it in each update:

_Fictitious example from 1.14.0 to 1.15.0 in which in between the image tag was previously updated to 2.16.0:_

Previous value:

releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.15.0.zip

New value:

releases/download/**v1.15.0**/enhanced-table-**1.15.0**_osd-**2.16.0**.zip

I've been looking at the examples but haven't seen anything "that complex": https://docs.renovatebot.com/modules/manager/regex/

**EDIT:** I have also the working regex https://regex101.com/r/VV1xcM/1

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by secustor** [comment]
Extract the main versions into an argument ( `ARG` ) and create a second custom manager which updates the opensearch version together with the base image.
